exercises/practice/etl/.docs/instructions.md

```diff
@@ -23,18 +12,16 @@ The old system stored a list of letters per score:
 - 8 points: "J", "X",
 - 10 points: "Q", "Z",
 
-The shiny new Scrabble system instead stores the score per letter, which makes it much faster and easier to calculate the score for a word.
-It also stores the letters in lower-case regardless of the case of the input letters:
+This needs to be changed to store each individual letter with its score in a one-to-one mapping.
 
 - "a" is worth 1 point.
 - "b" is worth 3 points.
 - "c" is worth 3 points.
 - "d" is worth 2 points.
-- Etc.
-
-Your mission, should you choose to accept it, is to transform the legacy data format to the shiny new format.
+- etc.
```

Comment:
Nit-pick here really, but the quotes are capitalization, the literal `a` is a quote of a literal `a`.  So the `E` should be capitalized, as the start of a sentence for the bullet, or the periods should be removed and a colon used on line 25 (the new line 15).  The colon was incorrect prior, given the periods ending the sentence, but not a big enough irritate to change, but with the changes coming in, we can now correct it without churn.

I will approve it anyway, since it is a minor thing, and I think many people would not notice.  As an exercise tangentially about languages, though, it is likely something that someone with language as an interest would notice.

Reply:
Since this is coming from a sync, it is really an upstream issue, and the change has already effected the other tracks, so I am going to merge it now.